Libplanet/Net/Swarm.cs

```diff
@@ -641,8 +641,13 @@ public void BroadcastTxs(IEnumerable<Transaction<T>> txs)
                                 ).GetValueOrDefault(a);
                         }
 
-                        IReadOnlyList<ActionEvaluation<T>> evaluations =
-                            block.Evaluate(DateTimeOffset.UtcNow, stateGetter).ToImmutableList();
+                        List<ActionEvaluation> evaluations =
+                            block.Evaluate(DateTimeOffset.UtcNow, stateGetter).ToList();
+                        if (!(_blockChain.Policy.BlockAction is null))
```

Comment:
It would be more straight-forward:

```suggestion
                        if (_blockChain.Policy.BlockAction is IAction)
```